python/src/LinearModelStepwiseAlgorithm_doc.i.in

```diff
@@ -37,6 +37,34 @@ and the :math:`n`-dimensional one :math:`\vect{X} = (X_i)_{i \leq n}` writes as
 
 where :math:`\epsilon` is the residual, supposed to follow the standard Normal
 distribution, :math:`\phi_i` the i-th element of the basis.
+
+
+Examples
+--------
+>>> import openturns as ot
+>>> '''Definition of the data set'''
```

Comment:
I think you want to use comments instead of string as this line does not do anything

Reply:
I do not necessarily agree as the string comment does indeed appear in the compiled html page. I have however transformed the string comments into standard text in order to make it more coherent with the rest of the API documentation, this change will be included in the next commit.

Reply:
the CI does not agree either ^^